Automattic/jetpack
# Jetpack ai search UX fixes, rename

This PR adds a series of UX improvements pointed in call for testing.
 
<img width="1227" alt="Zrzut ekranu 2023-09-28 o 13 58 54" src="https://github.com/Automattic/jetpack/assets/3775068/0c362621-837d-4654-b6a3-918a102471c2">


## Proposed changes:


 - Renames the block
 - Adds changes from #32945
 - Fixes #33332 
 - Fixes #33334 
 - Fixes issues with it seeming like you can do multiple submits at once.


## Does this pull request change what data or activity we track or use?
NO

## Testing instructions:
* Inspect the sidebar, confirm new icon and name
* Toggle copy button / references / feedback visibility
* Publish
* Use on frontend, see that hidden elements are hidden
* Use copy button
* Paste somewhere, observe no HTML in output
* Try and submit another question while one is rendering
